library/Translate/src/Juvix/Contextify/Passes.hs

```diff
@@ -9,25 +14,28 @@ import qualified Juvix.Contextify.InfixPrecedence.ShuntYard as Shunt
 import Juvix.Library
 import qualified Juvix.Library.NameSymbol as NameSymbol
 import qualified Juvix.Sexp as Sexp
+import qualified Juvix.Sexp.Structure.CoreNamed as StructureCore
 import qualified Juvix.Sexp.Structure.Frontend as Structure
 import Juvix.Sexp.Structure.Lens
-import qualified Juvix.Sexp.Structure.Transition as Structure
 import qualified StmContainers.Map as STM
 
-resolveModule ::
-  ExpressionIO m => Env.SexpContext -> m Env.SexpContext
-resolveModule context =
-  Env.passContextSingle context (\x -> x == ":atom" || x == Structure.nameOpenIn) openResolution
-
-inifixSoloPass ::
-  Expression m => Env.SexpContext -> m Env.SexpContext
-inifixSoloPass context =
-  Env.passContextSingle context (== Structure.nameInfix) infixConversion
+--------------------------------------------------------------------------------
+-- Environmental Constraint Types
+--------------------------------------------------------------------------------
 
 type ExpressionIO m = (Env.ErrS m, Env.HasClosure m, MonadIO m)
 
 type Expression m = (Env.ErrS m, Env.HasClosure m)
 
+--------------------------------------------------------------------------------
+-- Open Transformation
+--------------------------------------------------------------------------------
+
+resolveModule ::
```

Comment:
Can we add a brief description?

Reply:
Will do in #895! Should have done that a while ago!